### ISSUE TITLE
WARNING: An illegal reflective access operation has occurred

### ISSUE THREAD

**Issue by kenquiros64** [jfoenixadmin/JFoenix#1125]
I had this warning when running a project

WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.jfoenix.adapters.ReflectionHelper (file:/Users/kenneth/.m2/repository/com/jfoenix/jfoenix/9.0.10/jfoenix-9.0.10.jar) to method java.lang.reflect.AccessibleObject.setAccessible0(boolean)
WARNING: Please consider reporting this to the maintainers of com.jfoenix.adapters.ReflectionHelper
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

**Comment by iregon** [comment]
Same here:
````
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.jfoenix.adapters.ReflectionHelper 
(file:/C:/Users/Alessandro/.m2/repository/com/jfoenix/jfoenix/9.0.10/jfoenix-9.0.10.jar) to method java.lang.reflect.AccessibleObject.setAccessible0(boolean)
WARNING: Please consider reporting this to the maintainers of com.jfoenix.adapters.ReflectionHelper
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
````

Java version: 11.0.4
OpenJFX version: 13 (tested also with 11, 12 and 14)
JFoenix version: 9.0.10

**Comment by robskaar** [comment]
Im getting the same

WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.jfoenix.adapters.ReflectionHelper (file:/C:/xxx/xxx/xxx/xxx/xxx) to method java.lang.reflect.AccessibleObject.setAccessible0(boolean)
WARNING: Please consider reporting this to the maintainers of com.jfoenix.adapters.ReflectionHelper
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

**Comment by amiracam** [comment]
yep, not much more to say , but that the last line has me really worried :)

**Comment by hamzab70** [comment]
Getting the same error with JDK 11.0.9, OpenFX 11 and Jfoenix 9.0.10

```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.jfoenix.adapters.ReflectionHelper (file: ./jfoenix-9.0.10.jar) to method java.lang.reflect.AccessibleObject.setAccessible0(boolean)
WARNING: Please consider reporting this to the maintainers of com.jfoenix.adapters.ReflectionHelper
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

**Comment by c-d0wd** [comment]
I have encounter this same issue. However, with JDK 16 this illegal access operation is denied as stated in the last warning
`WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.jfoenix.adapters.ReflectionHelper (file:/C:/Users/camerond/.m2/repository/com/jfoenix/jfoenix/9.0.10/jfoenix-9.0.10.jar) to method java.lang.reflect.AccessibleObject.setAccessible0(boolean)
WARNING: Please consider reporting this to the maintainers of com.jfoenix.adapters.ReflectionHelper
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release`
I had to change back to JDK 15 to get my application to run.

**Comment by louhy** [comment]
See https://github.com/sshahine/JFoenix/issues/1170
